extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -58,6 +58,8 @@ const EXIT_CURVE_ = bezierCurve(0.4, 0, 0.2, 1);
 /** @private @const {!../../../src/curve.CurveDef} */
 const PAN_ZOOM_CURVE_ = bezierCurve(0.4, 0, 0.2, 1.4);
 
+/** @private @const {!number} */
```

Comment:
nit: Primitive types like `number` don't need the `!` because they are non-nullable by default